src/SConscript

```diff
@@ -116,6 +116,7 @@ if GetOption('enable_tests') or GetOption('enable_benchmarks'):
     cenv.MergeVars(tool_env)
     cenv.MergeVars(test_env)
     cenv.Append(CPPDEFINES=('ROC_MODULE', 'roc_test'))
+    ubsan_options = cenv.Object('library/src/ubsan_options.cpp')
```

Comment:
It's better to add it to build only when sanitizers are enabled during build.

Also, the name is a bit confusing to me, I'd rename it to something like `sanitizer_options_obj` or `sanitizer_obj` or similar because "options" suggest that the variable contains some flags or settings, and actually it contains an object file.